include/ocpp/v201/charge_point.hpp

```diff
@@ -78,16 +82,33 @@ struct Callbacks {
     std::function<void(const std::optional<const int32_t> evse_id, const ResetEnum& reset_type)> reset_callback;
     std::function<void(const int32_t evse_id, const ReasonEnum& stop_reason)> stop_transaction_callback;
     std::function<void(const int32_t evse_id)> pause_charging_callback;
-    ///
-    /// \brief Change availability of charging station / evse / connector.
-    /// \param request The request.
-    /// \param persist True to persist the status after reboot.
-    ///
-    /// Persist is set to 'false' if the status does not need to be stored after restarting. Otherwise it is true.
-    /// False is for example during a reset OnIdle where first an 'unavailable' is sent until the charging session
-    /// stopped. True is for example when the CSMS sent an 'inoperative' request.
-    ///
-    std::function<void(const ChangeAvailabilityRequest& request, const bool persist)> change_availability_callback;
+
+    // Availability changing callbacks
+    // These callbacks execut the state transition on the charging station (e.g. enabling/disabling EVSEs)
+    // libocpp will call one callback for every component whose state changed - for example,
+    // if an EVSE is disabled, a callback will be called once for the EVSE, and once for each disabled connector.
+    // If a callback is not provided, libocpp will simply ignore it - however, at the very least the connector callback
+    // must be provided.
+
```

Comment:
The connector callback is mandatory because any charging station would need at least that to be able to respond to an OCPP ChangeAvailabilityRequest from the CSMS. Making it optional is quite easy, however making it mandatory could reduce confusion for new users of the library.